### PR TITLE
Batch norm docs -> r1.1

### PR DESCRIPTION
**Pull request by nealwu**
Fixed up the documentation for tf.contrib.layers.batch_norm and tf.layers.batch_normalization explaining how to include the update ops in the train_op. Also simplified the existing tf.layers.batch_normalization documentation to the least error-prone option (adding to the train_op).

PiperOrigin-RevId: 156609483

See https://github.com/tensorflow/tensorflow/commit/675f0f9073e9e6ac6df8018e972f2b38f2c4c23c for the same change in master.

### CODE REVIEW

**Comment by gunan** [comment]
We probably wont touch this branch anymore, as 1.2 is around the corner.
Is it OK if we only cherrypick this into r1.2?

**Comment by jhseu** [comment]
Yes, please recreate pulling this change into r1.2.
